Components/Atoms/NotificationsState.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
```

Comment:
do you need to import React and useState here? I don't see it being used here

Reply:
good catch